frontend/tailwind.config.js

```diff
@@ -7,6 +7,12 @@ export default {
   darkMode: 'class',
   theme: {
     extend: {
+      fontSize: {
+        '22': '22px',
+      },
+      screens: {
+        'custom': '258px',
+      },
```

Comment:
Why this? The present convention for mobile screens default classnames and for tablet and above sm: classes

Reply:
I kept this because when the screen width decrease less than equal custom (258px) 'continue with Google' and 'continue with Github' link btns start shrink so instead of displaying this, I thought to display only 'Google' and 'Github'.

Reply:
That's really nice thought, highly appreciated. 
<img width="522" alt="image" src="https://github.com/krishnaacharyaa/wanderlust/assets/116620586/cb0abc34-328a-44a5-a5b4-203719d35e5b">

But, I would say, we might not have to handle that case, because no screen is really within that 258px range, the above is iphone SE, I would say smallest screen in the mobile industry which someone could use. And ideal mobile screen wouldn't be that small, if you google the standard responsive mobile design, they would say greater than 360, so I think we don't have to consider this use case, because it is nearly 99.9% not valid :)
But good corner case I would say. Rest looks good to me, can we kindly aim at the sign up page now...

Reply:
> That's really nice thought, highly appreciated. <img alt="image" width="522" src="https://private-user-images.githubusercontent.com/116620586/320217072-cb0abc34-328a-44a5-a5b4-203719d35e5b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI0MjA1NjAsIm5iZiI6MTcxMjQyMDI2MCwicGF0aCI6Ii8xMTY2MjA1ODYvMzIwMjE3MDcyLWNiMGFiYzM0LTMyOGEtNDRhNS1hNWI0LTIwMzcxOWQzNWU1Yi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDA2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQwNlQxNjE3NDBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lN2RiM2QzNjE0N2I1MDVkY2VlMmM2MDc2NzY1N2FjNDdiZmY3MWI1Y2QxNmNjZjY0MjM1ZjI3NDUyOWQ1M2M0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.aRfvXhskIIdNlrnTyUN6VwK_aYqS63ADXRdZeoUyZn8">
> 
> But, I would say, we might not have to handle that case, because no screen is really within that 258px range, the above is iphone SE, I would say smallest screen in the mobile industry which someone could use. And ideal mobile screen wouldn't be that small, if you google the standard responsive mobile design, they would say greater than 360, so I think we don't have to consider this use case, because it is nearly 99.9% not valid :) But good corner case I would say. Rest looks good to me, can we kindly aim at the sign up page now...

Thank you for valuable insights and review.

Okay, lets move ahead mentor.